cmd/istio-cni/main_test.go

```diff
@@ -216,6 +217,18 @@ func TestCmdAddTwoContainers(t *testing.T) {
 	}
 }
 
+func TestCmdAddTwoContainersWithoutSideCar(t *testing.T) {
+	defer resetGlobalTestVariables()
+
+	testAnnotations = nil
```

Comment:
I think it'd be better to do `testAnnotations = map[string]string{}` just like in the `resetGlobalTestVariables` function.

Reply:
Actually, it'd be much better to do `delete(testAnnotations, sidecarStatusKey)` instead.

Reply:
That makes a lot more sense! Thank You. I have commited the change 👍